oras/defaults.py

```diff
@@ -48,3 +48,6 @@ class registry:
 blank_config_hash = (
     "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a"
 )
+
+# ModelPack annotations, refer to https://github.com/modelpack/model-spec/blob/main/specs-go/v1/annotations.go
+annotation_filepath = "org.cnai.model.filepath"
```

Comment:
```suggestion
annotation_filepath = "org.cncf.model.filepath"
```

to align with latest since Sandboxing.

Ref: https://github.com/modelpack/model-spec/blob/d96289821ac99a777baaec74c8f3ee2625255adc/docs/annotations.md?plain=1#L9